### PR TITLE
Document some BSP stuff

### PR DESCRIPTION
**Pull request by csauve**
Here are some notes I've kept from BSP research. I could write more but don't want to keep this info stashed away for too long.

### CODE REVIEW

File: src/content/blam/tags/readme.md [hunk 1]
```diff
@@ -13,7 +12,7 @@ Tags are the fundamental unit of resources which comprise a [map][]. They come i
   </figcaption>
 </figure>
 
-Tags are the fundamental unit of resources which comprise a [map][]. They come in many different types, each with a predefined structure and data _fields_. Tags can reference other tags, forming a tree-like structure of all resources necessary to compile a map.
+Tags are the fundamental unit of resources which comprise a [map][]. They come in many different types (sometimes called tag _classes_ or _groups_), each with a predefined structure and data fields. Tags can reference other tags by their _tag path_, forming a tree-like structure of all resources necessary to compile a map.
```
Comment (csauve) [inline src/content/blam/tags/readme.md:15]: Just cleaning up a duplicate intro paragraph from a lazy earlier merge conflict resolution.

File: src/content/games/h1/games.dot [hunk 2]
```diff
@@ -102,8 +102,9 @@ digraph G {
   mac_204 -> halomd;
 
   # cea and mcc
-  pc_gold -> ann;
+  pc_gold -> ann [label="Campaign"];
   ann -> mcc_xbone;
+  pc_gold -> mcc_xbone [label="Multiplayer"];
```
Comment (csauve) [inline src/content/games/h1/games.dot:107]: Adds a couple lines to the game evolution diagram showing that MCC came from CEA's MP and PC's campaign.

Comment (gbMichelle) [reply src/content/games/h1/games.dot:107]: I think it'd be more accurate to specify that MCC's campaign came from CEA, and CEA came from PC

Comment (csauve) [reply src/content/games/h1/games.dot]: Updated...

like this?
tmp-pr

**Pull request merged**